service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -58,6 +58,9 @@ private final Respondent getFirstRespondent() {
     }
 
     private final Proceeding proceeding;
+
+    @NotNull(message = "Enter the solicitor's full name" + VALIDATION_SEPARATOR + "Enter the solicitor's email")
```

Comment:
I think the case data level validation is normally "you need to add details for solicitor" and then the Solicitor level validation is a "you need to fill in these fields"

Reply:
Usually yes but it wasn't mentioned on the ticket so I kept it consistent.

Reply:
Confirm with Mia, for consistency we should really have - 

In the applicant section :
- You need to add details to applicants
- You need to add details to solicitor (or similar)

Reply:
After discussion with Mia, have made it consistent with the rest of the messages when the event hadn't been accessed